src/library/trackcollectionmanager.cpp

```diff
@@ -84,6 +85,11 @@ TrackCollectionManager::TrackCollectionManager(
                 this,
                 &TrackCollectionManager::libraryScanFinished,
                 /*signal-to-signal*/ Qt::DirectConnection);
+        connect(m_pScanner.get(),
+                &LibraryScanner::scanSummary,
+                this,
+                &TrackCollectionManager::libraryScanSummary,
+                /*signal-to-signal*/ Qt::DirectConnection);
```

Comment:
What's the meaning of `signal-to-signal`?

Reply:
I think this pattern has been used to indicate we don't connect to a slot but to a signal (signal forward essentially)